Mapbox-10-Prototype-2025-Info.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this actually needs to be added to the main info.plist. My project isn't seeing this key when I run it.

Reply:
It seems like when one creates a new SwiftUI project in XCode, there is not a Info.plist by default.  I found some threads about it: https://forums.developer.apple.com/forums/thread/727969.  It seems like there are bunch of different ways to add a plist though, do you know which one we should do?  Feel free to commit to this branch if you can get it working, otherwise we can pair on it later today.

Reply:
Ben and I talked about this in person.  This file was automatically generated when I added the mapbox token here:

<img width="857" alt="image" src="https://github.com/user-attachments/assets/b2b5fd98-801b-4e85-b963-6c0e5d31e469">

Ben had to add the token in the same place.  Not sure why it didn't work when the repo was pulled.